packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -140,6 +141,11 @@
   width: rem-override(15rem);
 }
 
+.usa-textarea {
+  height: rem-override(10rem);
+  resize: inherit;
```

Comment:
I just want to double check that `resize: inherit;` should go int this Formation override stylesheet. When this stylesheet is eventually removed, will the resize functionality still work?